source/publishing_data/05_processing_data/processors/retrieve_administrative_divisions.rst

```diff
@@ -31,8 +31,11 @@ To set the parameters of the Retrieve Administrative Divisions processor, follow
   * * Geo Point
     * Field which contains the geo point the processor will use to retrieve the administrative divisions
     * yes
+  * * Country
+    * The country on which we want to look for administrative divisions. Please note that the default 'World' country is a special choice: the processor will try to get back the divisions for the given level (next parameter) without being limited to any country, but that it is an hazardous operation since administrative divisions are not homogenized accross countries.
```

Comment:
across*
Also the wording "it is an hazardous operation" makes it scary. It should just be a warning note